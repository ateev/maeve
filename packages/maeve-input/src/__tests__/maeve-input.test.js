import React from 'react';
import { shallow } from 'enzyme';
import MaeveInput from '../maeve-input.jsx';

test('render maeve input', () => {
  const component = shallow(
    <MaeveInput />
  );
  expect(component).toMatchSnapshot();
});