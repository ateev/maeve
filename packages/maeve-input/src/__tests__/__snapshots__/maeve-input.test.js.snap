// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render maeve input 1`] = `
<div
  className="maeve-input"
>
  <label />
  <input
    onChange={[Function]}
    onFocus={[Function]}
    type="text"
    value=""
  />
</div>
`;
